src/app/exam/score/ExamScorePage.js

```diff
@@ -33,30 +31,45 @@ const ExamScorePage = () => {
         return <Spinner/>;
     }
 
+    const onButtonExportCSVClick = (examId) => {
+        examService.getExamTranscriptCsvFile(examId)
+        .then((res) => {
+            const url = window.URL.createObjectURL(new Blob([res.data]));
+            const link = document.createElement('a');
+            link.href = url;
+            link.setAttribute('download', 'ExamTranscript.csv');
+            link.click();
+        });
+    };
+
     return (
         <div className="exam-score">
-            <div className="font-poppins" style={{paddingTop: "20px", paddingBottom: "150px"}}>
-                <div className="mt-2" style={{display: "flex", justifyContent: "center"}}>
```

Comment:
Extract them to `<div className="exam-score">`.
These `<div>` tags are used for setting CSS, so we can extract them to one `<div>`.